src/posthog-core.ts

```diff
@@ -1733,6 +1768,20 @@ export class PostHog {
         return name
     }
 
+    _isIdentified(): boolean {
+        return (
+            this.persistence?.get_user_state() === 'identified' ||
+            this.sessionPersistence?.get_user_state() === 'identified'
+        )
+    }
+
+    _hasPersonProcessing(): boolean {
+        return !(
+            this.config.__preview_process_person === 'never' ||
+            (this.config.__preview_process_person === 'identified_only' && !this._isIdentified())
+        )
```

Comment:
```suggestion
        return (
            this.config.__preview_process_person === 'always' ||
            (this.config.__preview_process_person === 'identified_only' && this._isIdentified())
        )
```

is this easier to parse as a positive condition?

Reply:
(you could change the type definition so that you can have a switch case on the options if you're worried about future changes breaking it 🤷 )

Reply:
I'm just being super defensive here, so that any unexpected value makes this return true.

Also so I don't have to change a bunch of the old-style tests, which don't get default config values, and therefore have `__preview_process_person` as undefined (despite this being invalid in the type system).